### PR TITLE
Minor changes to compactable error message template

### PR DESCRIPTION
**Pull request by Manno15**

### CODE REVIEW

File: server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java [hunk 1]
```diff
@@ -76,7 +76,8 @@ public Files(SortedMap<StoredTabletFile,DataFileValue> allFiles,
       var compactingFiles =
           compacting.stream().flatMap(job -> job.getFiles().stream()).collect(Collectors.toSet());
       Preconditions.checkArgument(this.allFiles.containsAll(compactingFiles),
-          "Compacting not in set of all files %s %s", this.allFiles, compactingFiles);
+          "Compacting not in set of all files %s, compacting files %s", this.allFiles,
```
Comment (DomGarguilo) [inline server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java:79]:
```suggestion
          "Compacting not in set of all files: %s, compacting files: %s", this.allFiles,
```

File: server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java [hunk 2]
```diff
@@ -76,7 +76,8 @@ public Files(SortedMap<StoredTabletFile,DataFileValue> allFiles,
       var compactingFiles =
           compacting.stream().flatMap(job -> job.getFiles().stream()).collect(Collectors.toSet());
       Preconditions.checkArgument(this.allFiles.containsAll(compactingFiles),
-          "Compacting not in set of all files %s %s", this.allFiles, compactingFiles);
+          "Compacting not in set of all files: %s, compacting files: %s", this.allFiles,
+          compactingFiles);
```
Comment (milleruntime) [inline server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java:80]: The message itself is still confusing to me. How about this?
```suggestion
          "Compacting files %s not in set of all files: %s", compactingFiles,
          this.allFiles);
```

Comment (Manno15) [reply server/tserver/src/main/java/org/apache/accumulo/tserver/compactions/Compactable.java]: That does sound better

**Review by DomGarguilo — APPROVED** [review]

**Comment by milleruntime** [comment]
Thanks @Manno15 !

**Pull request merged**